### PR TITLE
Conceptualization updates

### PR DESCRIPTION
**Pull request by milstan**
Seveal updates reflecting our recent discussions. Please review.

### CODE REVIEW

**Comment by milstan** [comment]
ping @m4rk055 could you pelase review this?

File: refs/high-level-conceptualization/README.md [hunk 1]
```diff
@@ -109,12 +116,27 @@ States concern different qualities, such as (not limited to):
...
 Also, rights have states, meaning that in a particular state, a particular type of rights request MUST be rejected. For example, during mandatory keeping a DELETE request from the user MUST be rejected.
 
 > _The information provided here is just for illustrative purposes and does not imply any definitive naming or semantics of actual states._
 
+## DATA CAPTURE – VERSIONING
+
+Data Capture Fragments are subject to versioning. A particuler data fragment can evolve as a result of:
+- **Changes** by Data Subject during submission
+- **Rights Requests** by Data Subject asking to modify the data
+- **Modifications** by Data Consumers and DPOs (when the new data is collected offline from the Data Subject)
```
Comment (m4rk055) [inline refs/high-level-conceptualization/README.md:136]: Also modifications BY data subjects not related to a rights request, if the access to data is provided through the UI (changing the phone number on a user's profile page, request for additional data by data consumer, etc.).

**Review by m4rk055 — CHANGES_REQUESTED** [review]

File: refs/high-level-conceptualization/README.md [hunk 2]
```diff
@@ -147,3 +169,5 @@ Privacy, in a psychological sense, equals the freedom of an individual to engage
 Confidentiality is enforced by policy and by encryption. Control is enforced by law. blindnet allows software systems to handle both.
 
 However, there must be a separation between the two allowing a developer to use blindnet only for one of these two purposes, and use an alternative solution for the other. E.g., a developer can use a non-encrypted web form to capture data, manage his own access to it, and still use blindnet for managing consent, and submitter rights requests.
+
+See [Bridge Pattern, Gang of Four](https://refactoring.guru/design-patterns/bridge).
```
[the remark this answers is not in the document]
Comment (noelmace) [reply refs/high-level-conceptualization/README.md:173]: Alternative (approximative) proposal: "we recommend following the Bridge Pattern when implementing such a system"
Depending on what @milstan had in mind here.

File: refs/high-level-conceptualization/README.md [hunk 3]
```diff
@@ -43,12 +43,18 @@ Examples include: a file and a comment explaining it; a set of fields composing
...
 The granularity can also allow, when needed, to the data consumers to accept or reject certain fragments for reasons of readability/conformity, and make the submitter submit them again.
 
-It may also favor incremental data submission (submit par of the for today, and another part some other day when the submitter collects more data), as well as partial modification and data update focused only on particular fragments.
+It may also favor incremental data submission (submit part of the form today, and another part some other day when the submitter collects more data), as well as partial modification and data update focused only on particular fragments.
 
 <img width="300" alt="Checklist" src="./img/checklist.png">
 
+Capture Fragments can be subject to validations:
+- **a priori** using automatic validators (like checking the format of an e-mail adress, or extension of a file)
+- **a posteriori** using human input to vlaidate the conformity of submitted data with what is expected
```
[the remark this answers is not in the document]
Comment (noelmace) [reply refs/high-level-conceptualization/README.md:56]:
```suggestion
- **a posteriori** using human input to validate the conformity of submitted data with what is expected
```

File: refs/high-level-conceptualization/README.md (hunk 1 above)
Comment (milstan) [inline refs/high-level-conceptualization/README.md:136]:
```suggestion
- **Modifications** by Data Consumers (e.g., within modificaiton requests, or through common featues for user profile modification, or any other way) and DPOs (when the new data is collected offline from the Data Subject)
```

**Comment by milstan** [comment]
I've addressed the change. @noelmace feel free to do final revision and merge if OK.

File: refs/high-level-conceptualization/README.md [hunk 4]
```diff
@@ -147,3 +169,5 @@ Privacy, in a psychological sense, equals the freedom of an individual to engage
 Confidentiality is enforced by policy and by encryption. Control is enforced by law. blindnet allows software systems to handle both.
 
 However, there must be a separation between the two allowing a developer to use blindnet only for one of these two purposes, and use an alternative solution for the other. E.g., a developer can use a non-encrypted web form to capture data, manage his own access to it, and still use blindnet for managing consent, and submitter rights requests.
+
```
Comment (milstan) [inline refs/high-level-conceptualization/README.md:172]:
```suggestion
Composition of the components of the system is inspired by the bridge design pattern.
```

File: refs/high-level-conceptualization/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (milstan) [reply refs/high-level-conceptualization/README.md:173]: done

File: refs/high-level-conceptualization/README.md (hunk 1 above)
Comment (milstan) [inline refs/high-level-conceptualization/README.md]:
```suggestion
- **Modifications** by Data Consumers and DPOs (when the new data is collected offline from the Data Subject, or a modification directly made by the Data Subject through a user interface).
```

Comment (milstan) [inline refs/high-level-conceptualization/README.md]:
```suggestion
- **Modifications** by Data Subjects (directly throught user interface) or by Data Consumers and DPOs (when the new data is collected offline from the Data Subject)
```

**Comment by milstan** [comment]
changes made. @m4rk055 see if OK and @noelmace please merge

**Review by m4rk055 — APPROVED** [review]

**Comment by milstan** [comment]
@noelmace please merge now. I need this for other updates

**Pull request merged**